specification/dnc/resource-manager/Microsoft.DelegatedNetwork/preview/2020-08-08-preview/orchestrators.json

```diff
@@ -0,0 +1,562 @@
+          "enum": [
+            "Kubernetes"
+          ],
+          "x-ms-enum": {
+            "name": "OrchestratorKind",
+            "modelAsString": true
+          }
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The resource tags."
+        }
+      },
+      "required": [
+        "kind"
+      ],
+      "x-ms-azure-resource": true
+    },
+    "OrchestratorResourceUpdateParameters": {
+      "description": "Parameters for updating a resource.",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The resource tags."
+        }
+      }
+    },
+    "Orchestrators": {
+      "description": "An array of OrchestratorInstance resources.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Orchestrator"
+          },
+          "description": "An array of OrchestratorInstance resources."
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "Orchestrator": {
+      "description": "Represents an instance of a orchestrator.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/OrchestratorResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/OrchestratorResourceProperties",
+          "description": "Properties of the provision operation request."
+        }
+      }
+    },
+    "OrchestratorResourceProperties": {
+      "description": "Properties of orchestrator",
+      "type": "object",
+      "properties": {
+        "resourceGuid": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource guid."
+        },
+        "provisioningState": {
+          "type": "string",
+          "enum": [
+            "Deleting",
+            "Succeeded",
+            "Failed",
+            "Provisioning"
+          ],
+          "x-ms-enum": {
+            "name": "OrchestratorInstanceState",
+            "modelAsString": true
+          },
+          "readOnly": true,
+          "description": "The current state of orchestratorInstance resource."
+        },
+        "orchestratorAppId": {
+          "description": "AAD ID used with apiserver",
+          "type": "string"
+        },
+        "orchestratorTenantId": {
+          "description": "TenantID of server App ID",
+          "type": "string"
+        },
+        "clusterRootCA": {
+          "description": "RootCA certificate of kubernetes cluster base64 encoded",
+          "type": "string"
+        },
+        "apiServerEndpoint": {
+          "description": "K8s APIServer url",
+          "type": "string"
+        },
+        "controllerDetails": {
```

Comment:
If the managed identity is associated with the orchestrator what is the dncAppId exposed in the controller still used for?  Can it be removed from that resource?

Reply:
explained above